### DISCUSSION TITLE
希望能将“稿件投诉”隐藏，或者放到更多选项里面。

### DISCUSSION THREAD

**Discussion by kethoven** [the1812/Bilibili-Evolved#4124] [功能建议 / Ideas]
视频的工具栏里面的“稿件投诉”几乎是一个很少使用的功能，而且同时使用稍后再看 快速收藏之后。就会出现错行，影响美观。
希望能将“稿件投诉”隐藏，或者放到更多选项里面。

**Comment by lovelyCARDINAL** [comment]
记笔记 要不要也放进下拉栏？

**Reply by kethoven** [reply]
> 记笔记 要不要也放进下拉栏？

我觉得都可以，如果可以自己设置的话。
记笔记倒算是一个偶尔用得到的功能，稿件投诉对我而言，基本上无用的功能。

**Reply by lovelyCARDINAL** [reply]
隐藏很容易 放进下拉栏就要写功能了 所以需要明确需求

**Reply by kethoven** [reply]
> 隐藏很容易 放进下拉栏就要写功能了 所以需要明确需求

对我而言，投诉无用，可以直接隐藏‘稿件投诉’。

只是如果能够将“投诉”和“笔记”可以分开设置隐藏，那么隐藏功能就已经非常充足了。

**Comment by Geeyun-JY3** [comment]
才发现原来我以前提过收进更多菜单：https://github.com/the1812/Bilibili-Evolved/issues/1076#issuecomment-738898732
也有其他人提过：https://github.com/the1812/Bilibili-Evolved/issues/1648#issuecomment-800242753
作者也好像回应过（？）：https://github.com/the1812/Bilibili-Evolved/issues/1648#issuecomment-802722739

**Comment by Geeyun-JY3** [comment]
一些情况下破站也会把「记笔记」和「稿件投诉」收进更多选项里（应该是根据宽度自适应的吧）。
以下是我的电脑在宽屏模式下的实测：
![image](https://user-images.githubusercontent.com/49396421/235439141-0f48ca0a-608f-4bad-aecf-67fa19bf8d25.png)

**Reply by lovelyCARDINAL** [reply]
那这是怎么触发的呢……

**Reply by Geeyun-JY3** [reply]
> 以下是我的电脑在**宽屏模式**下的实测
